docs/applications/ddns-updater.md

```diff
@@ -0,0 +1,15 @@
+# DDNS Updater
+
+Homepage: <https://github.com/qdm12/ddns-updater>
+
+DDNS Updater is a container to update DNS records periodically with WebUI for many DNS providers. Check the homepage for the list of 30+ providers it supports.
+
+## Usage
+
+Set `ddns_updater_enabled: true` in your `inventories/<your_inventory>/nas.yml` file.
+
+The DDNS Updater web interface can be found at <http://ansible_nas_host_or_ip:8001>.
+
+## Specific Configuration
+
+DDNS Updater requires editing its <docker_home>/ddns-updater/data/config.json file. Instructions can be found [here](https://github.com/qdm12/ddns-updater#Configuration).
```

Comment:
Users shouldn't have to manually meddle with stuff in `<docker_home>/ddns-updater/data/config.json` to get set up. Please add as a yaml variable and template in at deploy.

Reply:
There are 30+ DDNS providers that DDNS Updater supports. Editing the config.json file in the persistent data directory for each provider is very well documented on the dev's site. 
I was going to say that this is impossible to automate, but nothing's impossible. It just would require creating anywhere up to 31 template files since the fields in the json file would have some standard definitions overlap but also contain provider specific fields. On top of that the task itself would need to filter through many conditions based on the variables enabled to pick and generate from the template files. Again, not impossible but a lot of work. IMHO if someone can login to their any of their apps and do basic setup they edit one file in their Docker persistent data dir. Consider how complex and time consuming it is to setup Nextcloud vs editing one text file for this.

If you're hard fast on this, I'd recommend a compromise of getting the app pulled in as is so people can make use of it and then make it an issue to improve upon it.

Reply:
(When editing the scrutiny.md file, a lightbulb magically appeared above my head...)

To put it in perspective and drive it home: Count the number of times the AN app documentation says 

> edit your `inventories/<your_inventory>/nas.yml` file

![image](https://user-images.githubusercontent.com/14026600/185202260-c47cffb2-548c-4d81-9b04-444c05065765.png)

Reply:
It doesn't need to be that complex. Yaml is JSON, JSON is yaml. 

You could add it a variable, then use Ansible filters to transform into the correct format at template time. A quick Google will reveal lots of examples. There are other examples already in Ansible-NAS. 

A key tenet of Ansible-NAS is that it *automates* setup. If you have to run the playbook then immediately log in to fix something the playbook didn't do, then that tenet is broken (and why I hate nextcloud so much!)

Reply:
I get it. I know how to do it, I just didn't want to have to do it 31 times!

I'll make the 31 template files...